### PR TITLE
Updated 1st par with revisor demanding

### PR DESCRIPTION
**Pull request by lrsantos11**
Regarding, Issue #28, I had to include some references. Please revise and merge.

### CODE REVIEW

File: overview.tex [hunk 1]
```diff
@@ -1,25 +1,31 @@
...
+    to evaluate and compare the performance of a set of solvers $\Sset$ on a 
+    test set $\Pset$. It is presented as a graphic that shows the cumulative
+    distribution function of different solvers performances,
+    according to the  chosen cost metric, for a given set of problem instances.
+
+    For each problem $p \in \Pset$ and solver $s \in \Sset$, let $t_{ps}$
+    be the cost required to solve problem $p$ by solver $s$ and
+    \begin{align*} r_{ps} = \frac{t_{ps}}{\min\{t_{ps}: s \in \Sset\}}
+    \end{align*} be the performance ratio of solver $s$ for the problem $p$
+    when compared with the best performance by any solver on this problem.
```
Comment (abelsiqueira) [inline overview.tex]: Have you used some sort of automatic line breaker? This align shouldn' t be written like this.

**Comment by abelsiqueira** [comment]
In some places, there was only a change in the line break. I rather don't do this because it will conflict other merges.

**Comment by lrsantos11** [comment]
I think I changed something in my settings for another project, I forgot it  and it was carry over with this one.
Who should  resolve the conflicts?

**Comment by abelsiqueira** [comment]
You should. One way to do it is rebase and amend; another is to cherry-pick, and there should be others too. Either way, push the commit to the same branch and this PR will update automatically.

**Comment by abelsiqueira** [comment]
@lrsantos11, the problems created with new settings weren't fixed. The align and the extra line breaks are still there. You only merged my changes and pushed the merge commit. This fixes the conflict only.
Also, the history will look awful (see the newtork).

The good way to do it is to rebase the first commit to this master (keep a branch with the latest update to prevent any loss in your work), then, since you committed that wrong part with the line breaks, you have to amend your changes. You is to use `git reset HEAD~1 --soft` which will revert the rebased commit and leave the changes cached. Then modify the file, add, commit, push with `-f`.

**Comment by abelsiqueira** [comment]
Merged manually

**Pull request closed without merging**